gulpfile.js

```diff
@@ -69,6 +70,37 @@ var webpackConfig = {
   plugins: gulpPlugins
 };
 
+gulp.task('test-unit', function (done) {
+    new KarmaServer({
+        configFile: __dirname + '/karma.conf.js'
+    }, done).start();
```

Comment:
Yes, it is by design. In the test-unit task, I am keeping the karma server alive (karma option `singleRun`, defaulted to false.) The source and specs are watched and the tests re-run whenever changed. This task is intended for development rather than CI.

Inversely, the test-coverage explicitely sets singleRun to true and exists after having run the tests. This intended for CI rather than development.